### PR TITLE
Logic for Add avatar and comment

### PR DESCRIPTION
**Pull request by praveenkumartam2974**
Code Logic for Add avatar and comment

### CODE REVIEW

File: MusicAppService/controllers/userController.js [hunk 1]
```diff
@@ -116,4 +115,101 @@ async function getUsersById(req, res) {
...
+}
+
+//User can update comment for the song
+async function getUserCommentById(req, res) {
+  try {
+    const _id = req.params.id;
+    const userCommentData = await UserService.getUserCommentById(_id);
+    if (!userCommentData) {
+      res.status(200).json({
+        Message: "No Users"
```
Comment (Dineshks15) [inline MusicAppService/controllers/userController.js:194]: Hi @praveenkumartam2974 , Please change status code 200 -> 401

File: MusicAppService/controllers/userController.js [hunk 2]
```diff
@@ -116,4 +115,101 @@ async function getUsersById(req, res) {
...
+}
+
+//Gets User's avatar
+async function getUsersAvatarById(req, res) {
+  try {
+    const _id = req.params.id;
+    const userAvatarData = await UserService.getUsersById(_id);
+    if (!userAvatarData) {
+      res.status(200).json({
+        Message: "No Users"
```
Comment (Dineshks15) [inline MusicAppService/controllers/userController.js:151]: Hi @praveenkumartam2974  , Please change status code 200 -> 401.

File: MusicAppService/services/service.js [hunk 3]
```diff
@@ -343,6 +343,70 @@ async function getSongsByPlayListId(playlistId) {
...
+  try {
+    const userData = await Users.findById({ _id });
+    return userData.avatarUrl;
+  } catch (error) {
+    return error;
+  }
+}
+
+//add comment to playlist
+async function addComment(email, comment) {
```
Comment (Team2DemoApp) [inline MusicAppService/services/service.js:376]: controller is sending parameters _id, comment. Why service has email? From where this email will be set?
**const userComment = await UserService.addComment(
      _id,
      comment
    );**

File: MusicAppService/services/service.js [hunk 4]
```diff
@@ -343,6 +343,70 @@ async function getSongsByPlayListId(playlistId) {
...
+  try {
+    const playlist = await Playlist.find({
+      $where: "this.email === '" + email + "'"
+    });
```
Comment (Team2DemoApp) [inline MusicAppService/services/service.js:380]: what is the use of this playlist?

File: MusicAppService/services/service.js [hunk 5]
```diff
@@ -343,6 +343,70 @@ async function getSongsByPlayListId(playlistId) {
...
+    // const userData = await Users.getUserPlayList({ _id });
+    const comments = await Users.findByIdAndUpdate(
+      { email: email },
+      {
+        comment: comment
+      },
+      {
+        new: true
+      }
+    );
```
Comment (Team2DemoApp) [inline MusicAppService/services/service.js:390]: comment is in playlist schema, why find is done from users?

And, where is add happening?

**Review by Team2DemoApp — CHANGES_REQUESTED** [review]

**Comment by Team2DemoApp** [comment]
@praveenkumartam2974, PR cannot be merged, comments addressed in another PR.

**Pull request closed without merging**